### PR TITLE
chore : Guided Tour Page Deduplication

### LINKED ISSUE #22447: [Task]: Deduplicate components & Grayscale/theming support in Welcome Tour Page

### PR DESCRIPTION
**Pull request by rohitagarwal88**
> Pull Request Template
>
> Use this template to quickly create a well written pull request. Delete all quotes before creating the pull request.

## Description
Fixes #22447 

https://www.loom.com/share/b942553f00b445e4914be9f0988f1a8c

> Add a TL;DR when description is extra long (helps content team)

Fixes # (issue)
> if no issue exists, please create an issue and ask the maintainers about this first

Media
> A video or a GIF is preferred. when using Loom, don’t embed because it looks like it’s a GIF. instead, just link to the video

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Chore (housekeeping or task changes that don't impact user perception)
- This change requires a documentation update

- Manual
- Jest
- Cypress

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/pages/Editor/GuidedTour/Guide.tsx [hunk 1]
```diff
@@ -155,7 +128,7 @@ const Hint = styled.div`
   }
 
   .hint-text {
-    font-size: 16px;
+    font-size: 14px;
```
Comment (akash-codemonk) [inline app/client/src/pages/Editor/GuidedTour/Guide.tsx:131]: Text looks a bit small to me.

**Review by akash-codemonk — APPROVED** [review]

**Comment by laveena-en** [comment]
@rohitagarwal88 Issues with guided tour:

1. Appsmith logo is missing
image

2. Can we use a larger version version of the buttons to match prod
image

3. Sentence casing to be used for all buttons
4. Heading colour seems to be deviating from design. Shouldn’t it be `#364252`
5. Orange marker over run is missing
image
6. On deploying the app (final step), the `Congratulations...` section isn’t present
7. Can we increase the size of all text in the blue portion and the size of the `End Tutorial` button? It looks quite small when compared to release/prod

Clarifications:
There are several other issues with the guided tour like icon being missed, hover state looking off, overlap in the query section. Which PR would potentially cover this?

**Review by albinAppsmith — APPROVED** [review]

**Review by albinAppsmith — APPROVED** [review]

**Comment by laveena-en** [comment]
@rohitagarwal88 

1. `Proceed` button's spelling is incorrect in step 4
image

2. Second line to be aligned with the same left alignment as first
image

@vasanth-appsmith Please re-check and give a final sign-off from design.

**Pull request merged**